jujugui/static/gui/src/test/assets/chai.js

```diff
@@ -0,0 +1 @@
+../../../../../../node_modules/chai/chai.js
```

Comment:
lol - yeah these were symlinked so they needed to be updated for the new folder structure.